plugin.json

```diff
@@ -4,7 +4,7 @@
   "Name": "Everything",
   "Description": "Search Everything",
   "Author": "qianlifeng,orzfly",
-  "Version": "1.4.3",
```

Comment:
Hmm, how come this diff is showing 3 not 6

Reply:
Hmm I believe I haven't fetch before creating the branch.